network/src/memory_peer_store.rs

```diff
@@ -1,42 +1,34 @@
-use crate::peer_store::{Behaviour, PeerStore, Status};
+use crate::peer_store::{Behaviour, PeerStore, ReportResult, Score, ScoringSchema, Status};
 use crate::PeerId;
 use fnv::FnvHashMap;
 use libp2p::core::Multiaddr;
-use log::trace;
-use std::time::Instant;
-
-// peer_id -> addresses,
-// sort by score
-// addr -> peer_id
-// last report or updated_time
-const INITIALIZED_SCORE: u32 = 0;
+use log::{debug, trace};
+use std::time::{Duration, Instant};
 
 #[derive(Debug)]
 struct PeerInfo {
     addresses: Vec<Multiaddr>,
     last_updated_at: Instant,
-    score: u32,
+    score: Score,
     status: Status,
 }
 
 // NOTICE MemoryPeerStore is used for test environment only!!!
 pub struct MemoryPeerStore {
```

Comment:
we can add `Default` derive to simplify test code

Reply:
SQLite based peer store should be the default implementation when it's ready.